src/scripts/main.js

```diff
@@ -1,7 +1,37 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const container = document.createElement('container');
+
+  container.classList.add('notification');
+
+  if (type === 'success') {
+    container.classList.add('notification', 'success');
+  } else if (type === 'error') {
+    container.classList.add('notification', 'error');
+  } else if (type === 'warning') {
+    container.classList.add('notification', 'warning');
+  }
+
+  const messageTitle = document.createElement('h2');
+
+  messageTitle.classList.add('title');
+  messageTitle.textContent = title;
+
+  const descriptionElement = document.createElement('p');
+
+  descriptionElement.textContent = description;
+
+  container.style.top = posTop + 'px';
+  container.style.right = posRight + 'px';
+
+  document.body.appendChild(container);
```

Comment:
Consider appending the 'container' to the body before setting its position styles to avoid potential layout reflows.